web_ui/src/webworkers/intelligent-scissors.worker.ts

```diff
@@ -152,12 +152,12 @@ class IntelligentScissors {
     }
 }
 
-const waitForOpenCV = async (): Promise<boolean> => {
+const loadOpenCV = async (): Promise<boolean> => {
```

Comment:
For now i am slightly changing this method to prevent breaking existing tools. After i move them they will use the same approach as watershed or inferenceImage tools